browser.css

```diff
@@ -73,3 +73,19 @@ html.os-darwin #react-root ._246HcQqT {
 ._1gSEd2sj {
 	width: 100%;
 }
+
+/* change scrollbar style */
+::-webkit-scrollbar {
```

Comment:
None of these styles should be applied on OS X though, as it has a better system default scrollbar. You can use the OS specific styles to do this. Something like `html:matches(.os-linux, .os-win32) ::-webkit-scrollbar {`.